front/src/app/pages/group/GroupPage.tsx

```diff
@@ -81,15 +81,21 @@ const GroupPageContent = ({ group, results }: GroupWithResults) => {
   const filterResults = useCallback(
     (query: string) => {
       setDisplayedResults(
-        results.filter((displayedResult: SearchResultDto) =>
-          `${displayedResult.additionalInformation}
-            ${displayedResult.customizedName}
-            ${displayedResult.name}
-            ${displayedResult.address.city}
-            ${displayedResult.nafLabel} `
+        results.filter((displayedResult: SearchResultDto) => {
+          const appellationLabels = displayedResult.appellations
+            .map(({ appellationLabel }) => appellationLabel)
+            .join(" ");
+
+          return ` ${appellationLabels}
```

Comment:
il y a un espace vide ici, mais sinon ok pour moi

```suggestion
          return `${appellationLabels}
```